README.ja-jp.md

```diff
@@ -0,0 +1,265 @@
+
+# Form OCR Testing Tool
+
+Form Recognizer の改善にご協力ください。[Take our survey!](https://aka.ms/FR-HaTS-Survey)
+
+## Features Preview
+
+[![Build Status](https://dev.azure.com/msazure/Cognitive%20Services/_apis/build/status/microsoft.OCR-Form-Tools?branchName=master)](https://dev.azure.com/msazure/Cognitive%20Services/_build/latest?definitionId=118293&branchName=master)
+
+[Form Recognizer](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/) のオープンソースの Labeling Tool で、Form OCR Test Toolset (FOTT) の一部です。
+
+これはツールの `MAIN` ブランチです。これは、利用可能なすべての最新機能を含んでいます。プレビューのため、最も安定したバージョンでは**ありません**。
+
+このリポジトリの目的は、お客様が Microsoft Forms と OCR サービスを使用する際に、最新のツールをテストできるようにすることです。現在のところ、Labeling Tool がここで紹介する最初のツールです。ユーザーはフィードバックを提供し、独自のニーズを満たすために顧客固有の変更を行うことができます。Microsoft Azure Form Recognizer チームは、定期的にソースコードを更新する予定です。コントリビュートしたいと思われる方は、コントリビュートのセクションをご覧ください。
+
+**このツールの最新の GA バージョンをチェックしたい方は、こちらの[リンク]((https://github.com/microsoft/OCR-Form-Tools/tree/v2.0-General-Availability-Release))をご覧ください。**
+
+___
+
+FOTT の Labeling Tool は、TypeScript を使った React + Redux の Web アプリケーションです。このプロジェクトは [Create React App](https://github.com/facebook/create-react-app) でブートストラップしています。
+
+現在の Labeling Toolの機能 ([こちら](https://youtu.be/GRGs18lZJ1E?t=462)でショートデモが見れます)
+
+* PDF、JPEG、TIFF 形式のフォームに**ラベルを付ける**
+* [Form Recognizer](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/) でラベル付けされたデータを使ってモデルを**学習する**
+* 学習したモデルを使ってフォームを**予測／分析**をし、そのフォームから予測／分析した結果のキー・バリューを抽出
+
+## Getting Started
```

Comment:
suggestion: would you mind also translating the headings in this readme?

Reply:
Thanks! I had unconsciously missed it. I added the translation.